### PR TITLE
Messenger Track Kit

### PR DESCRIPTION
**Pull request by liach**
See CovertJaguar/Railcraft#1095

### CODE REVIEW

**Comment by hamzacanalt** [comment]
Dont send email for this thread please

Android için Outlook<https://aka.ms/ghei36> uygulamasini edinin


________________________________

File: assets/railcraft/lang/zh_CN.lang [hunk 1]
```diff
@@ -529,6 +529,8 @@ railcraft.gui.mag.glass.aspect=信号灯颜色是%s.
 railcraft.gui.mag.glass.aspect.dual=信号灯顶部颜色是%1$s底部颜色是%2$s.
 railcraft.gui.mag.glass.placedby=%1$s被%2$s放置于此.
 
+railcraft.gui.message.set=成功将信息设为:
```
Comment (3TUSK) [inline assets/railcraft/lang/zh_CN.lang:532]: 全角冒号

File: assets/railcraft/lang/zh_CN.lang [hunk 2]
```diff
@@ -1254,6 +1256,9 @@ track_kit.railcraft.railcraft_locomotive.tips=发动或停止机车\n§o§9- 通
 track_kit.railcraft.railcraft_missing.name=???
 track_kit.railcraft.railcraft_missing.tips=只是拿来占位的，这说明这个轨道出问题了
 
+track_kit.railcraft.railcraft_messenger.name=传信
```
Comment (3TUSK) [inline assets/railcraft/lang/zh_CN.lang:1259]: 发信？

Comment (liach) [reply assets/railcraft/lang/zh_CN.lang:1259]: 展示一个信息，就像一个指令链接，彩色字，翻译之类的。要改直接在分支上改吧。

**Comment by hamzacanalt** [comment]
________________________________

File: assets/railcraft/lang/zh_CN.lang (hunk 1 above)
Comment (liach) [reply assets/railcraft/lang/zh_CN.lang:532]: 呵呵了，我手机上改的，用搜狗的英文冒号

**Comment by liach** [comment]
@hamzacanalt Click that link in the email to mute this thread. You receive this since you watch this repository.

**Comment by liach** [comment]
Will merge since the main pull is merged.

**Pull request merged**